src/IssueViz/Editor/LocationTagging/IIssueLocationStore.cs

```diff
@@ -36,10 +36,16 @@ public interface IIssueLocationStore
         /// </summary>
         IEnumerable<IAnalysisIssueLocationVisualization> GetLocations(string filePath);
 
+        /// <summary>
+        /// Returns all the primary issues
+        /// </summary>
+        IEnumerable<IAnalysisIssueVisualization> Get();
```

Comment:
`GetIssues` might be a better name - more consistent with `GetLocations`, and it would be reasonable to expect a `IIssueLocationStore.Get` to return locations rather issues.